portfolio/src/main/webapp/script.js

```diff
@@ -13,16 +13,64 @@
 // limitations under the License.
 
 /**
- * Adds a random greeting to the page.
+ * Adds a random Bible verse to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+let bibleVerses = new Map();
```

Comment:
Consider in the future making this a const variable - I know you don't now because how you construct it, but think about how you could construct it differently such that it can be const. https://google.github.io/styleguide/jsguide.html#naming-constant-names.

I don't want to block your submission on that at the moment though!

Reply:
So I changed 'let' to 'const' and changed the variable name from 'bibleVerses' to 'BIBLE_VERSES'. Was that it?

Reply:
Yep! As per here https://google.github.io/styleguide/jsguide.html#naming-constant-names, that's sufficient. This convention notifies anyone using your code that BIBLE_VERSES should be immutable after initialization.

Reply:
Great then!